src/kibana/components/agg_types/controls/_values_list.js

```diff
@@ -122,9 +122,9 @@ define(function (require) {
           if (isNaN(num)) return INVALID;
 
           var list = $listGetter($scope);
-          var min = list[$scope.$index - 1] || 0;
-          var max = list[$scope.$index + 1] || 100;
-
+          var min = list[$scope.$index - 1] || $scope.valueBoundaries[0];
```

Comment:
Would you mind moving the valueBoundaries argument into the attributes of the element where `values-list` is specified?

Something like: 
```html
<input type="text" value-list="some.property" value-list-min="0" value-list-max="100">
```
and
```js
link: function ($scope, $el, attrs) {
  var min = _.parseInt(attrs.valueListMin);
  if (isNaN(min)) throw new TypeError('invalid value-list-min: ' + attrs.valueListMin);

  var max = _.parseInt(attrs.valueListMax);
  if (isNaN(max)) throw new TypeError('invalid value-list-max: ' + attrs.valueListMax);
}
```